frontend/package.json

```diff
@@ -6,7 +6,7 @@
     "node": ">=20.0.0"
   },
   "scripts": {
-    "all-checks": "npm run test && npm run lint && npm run ts:check && npm run build",
+    "all-checks": "npm run lint && npm run ts:check && npm run test && npm run build",
```

Comment:
@doug-s-nava I moved `npm run test` last since the test takes longest and it stops if the test fails (which has happened a couple of times).